### PR TITLE
Support: txt file support added

### PR DESCRIPTION
**Pull request by codePerfectPlus**
# Change log

1. txt file format is supported now for reading and converting into audio format
2. Reading function is moved to create_Json. now every file will be convert in JSON format first

### CODE REVIEW

**Comment by DrakeEntity** [comment]
Creating JSON format first will take some time first. it may take lots of time to big pdfs.

**Comment by codePerfectPlus** [comment]
> Creating JSON format first will take some time first. it may take lots of time to big pdfs.

It's required so we can have a common format.

File: audiobook/main.py [hunk 1]
```diff
@@ -45,15 +63,28 @@ def create_json_book(self, pdf_file_path, password=None):
...
+            book_dict, pages = self.pdf_to_json(input_file_path, password)
+        elif input_file_path.endswith(".txt"):
+            book_dict, pages = self.txt_to_json(input_file_path)
+        return book_dict, pages
+    
+    def save_audio(self, input_file_path, password=None):
+        self.file_check(input_file_path)
+        with open(input_file_path, "rb") as fp:
+            basename = os.path.basename(input_file_path).split(".")[0]
             os.makedirs(basename, exist_ok=True)
```
Comment (aayushi-droid) [inline audiobook/main.py:88]: you forget to get book_dict in save_audio function.

**Review by aayushi-droid — CHANGES_REQUESTED** [review]

File: audiobook/main.py (hunk 1 above)
Comment (codePerfectPlus) [reply audiobook/main.py:88]: Thanks for pointing out the issue

**Review by aayushi-droid — APPROVED** [review]

**Review by DrakeEntity — APPROVED** [review]

**Pull request merged**